app/controllers/relationships_controller.rb

```diff
@@ -33,7 +33,7 @@ def set_accounts
   end
 
   def set_relationships
-    @relationships = AccountRelationshipsPresenter.new(@accounts.pluck(:id), current_user.account_id)
+    @relationships = AccountRelationshipsPresenter.new(@accounts, current_user.account_id)
```

Comment:
Is the `@accounts` here (coming back from RelationshipFilter) guaranteed to always be an `Account` scope? If so, could limit the `select(..)` here as well

Reply:
It is guaranteed, but this controller does really different things to `/api/v1/accounts/relationships` and needs accounts in full, the `AccountRelationshipsPresenter` part is just a small part of it.